Smartling.API/Utils/LimitedConcurrencyLevelTaskScheduler.cs

```diff
@@ -0,0 +1,129 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Smartling.Api.Threading
```

Comment:
What do you think about namespace and folder name? Would like to make them in .net style.

Reply:
It is good